TimberAPI/AssetLoaderSystem/AssetSystem/CustomAssetBundle.cs

```diff
@@ -39,7 +39,7 @@ public void Load()
             {
                 if (_plugin.LoadingScene == SceneEntryPoint.Global)
                     return;
-                Console.WriteLine($"Asset: {FileName} was already loaded.");
+                Log.LogError($"Asset: {FileName} was already loaded.");
```

Comment:
I feel like this and the log in Unload() should be logged as warnings not errors